src/libutil/serialise.cc

```diff
@@ -107,6 +107,20 @@ std::string Source::drain()
 }
 
 
+size_t Source::drainAll(const size_t bufSize)
```

Comment:
The `bufSize` argument seems unnecessary.

Reply:
How do you know what `bufSize` is the most efficient?

Reply:
It would be nice if there was a `seek-forward()` function since the reading is really pointless here.

Reply:
You don't, but neither does the caller...

Reply:
Maybe `sendfile()` to `/dev/null`. Probably not worth it though (and not very portable).

Reply:
Okay. I'm now using the same 8192 as in drain()